e2e_test/udf/python.slt

```diff
@@ -0,0 +1,48 @@
+# Before running this test:
+#   python3 e2e_test/udf/test.py
+
+# TODO: check the service on creation
+
+# Create a function.
+statement ok
+create function int_42() returns int as 'http://localhost:8815' language arrow_flight;
+
+statement ok
+create function gcd(int, int) returns int as 'http://localhost:8815' language arrow_flight;
+
+# Create a function with the same name but different arguments.
+statement ok
+create function gcd(int, int, int) returns int as 'http://localhost:8815' language arrow_flight;
+
+# Create a function with the same name and arguments.
```

Comment:
What about creating a function with the same name as a build-in function? 🥵

Reply:
At present, the creation will succeed but the usage will still refer to the built-in version. I will fix it later. 🥵